SpectraLogic.SpectraRioBrokerClient/Model/IJob.cs

```diff
@@ -65,6 +73,14 @@ public interface IJob
         /// </value>
         JobType JobType { get; }
 
+        /// <summary>
+        /// Gets the last updated.
+        /// </summary>
+        /// <value>
+        /// The last updated.
```

Comment:
Not a sentence. Change to something like: `The date last updated` or `The timestamp of last update`